CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 0.1.10
+Released on January 17, 2023
+
+### Enhancements
+* For workflows that are triggered at the end of other workflows, we now allow changing the
```

Comment:
Nit: Add: "on the workflow settings dialog."

Reply:
done